src/sentry/api/endpoints/project_group_index.py

```diff
@@ -883,7 +883,7 @@ def delete(self, request, project):
         else:
             # missing any kind of filter
             return Response(
-                '{"detail": "You must specify a list of IDs for this operation"}', status=400
```

Comment:
The go-sentry-api tests were failing to unmarshal this response. Sentry was returning back "{\"detail\": \"You must specify a list of IDs for this operation\"}"